model/distro/distro.go

```diff
@@ -898,3 +898,15 @@ func GetHostCreateDistro(ctx context.Context, createHost apimodels.CreateHost) (
 func GetDistrosForImage(ctx context.Context, imageID string) ([]Distro, error) {
 	return Find(ctx, bson.M{ImageIDKey: imageID})
 }
+
+// GetImageIdFromDistro returns the imageID corresponding to the given distro.
+func GetImageIdFromDistro(ctx context.Context, distro string) (string, error) {
+	result, err := FindOne(ctx, bson.M{IdKey: distro})
```

Comment:
it might be nice to call the return value `d` (for distro)
```suggestion
	d, err := FindOne(ctx, bson.M{IdKey: distro})
```